packages/components/src/components/progress-bar/progress-bar.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import Text from '../text';
 
 // Todo: Create a utility type for getting a range of float number. ex: FloatRange<0, 1>
```

Comment:
@sergei-deriv does this PR also implies that we address the TODO comment? 🙂 if yes, we can make a TFloatRange type according to an example in https://catchts.com/range-numbers

Reply:
> @sergei-deriv does this PR also implies that we address the TODO comment? 🙂 if yes, we can make a TFloatRange type according to an example in https://catchts.com/range-numbers

I'm not sure, honestly :) But if you think I have to do this, I'll try. But for my PR I've got this type like this:
type TRangeFloatZeroToOne = React.ComponentProps<typeof ProgressBar>['value'];

Reply:
I think it’s out of this PR scope to handle the todo, But it will be nice if we add the generic @maryia-deriv suggested to `utils.d.ts` 👍🏻 Maybe in a separate card 🤔